### ISSUE TITLE
Build issue - retrieving data from teamcity.labkey.org

### ISSUE THREAD

**Issue by satsurae** [phnmnl/container-pwiz#9]
Hi,

Was attempting to build this image using:

`docker build --tag="phnmnl/pwiz--i-agree-to-the-vendor-licenses:latest" .`

 using `Docker version 18.03.1-ce, build 9ee9f40` on `Ubuntu 18.04 LTS`. However, I got the following error relating to it trying to get a response from teamcity:

```
Step 25/34 : RUN wget -O /tmp/pwiz.version https://teamcity.labkey.org/repository/download/bt36/`cat /tmp/pwiz.build`:id/VERSION?guest=1
 ---> Running in 404ce0877882
--2018-05-22 11:25:19--  https://teamcity.labkey.org/repository/download/bt36/574320:id/VERSION?guest=1
Resolving teamcity.labkey.org (teamcity.labkey.org)... 54.245.112.80
Connecting to teamcity.labkey.org (teamcity.labkey.org)|54.245.112.80|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2018-05-22 11:25:21 ERROR 404: Not Found.

The command '/bin/sh -c wget -O /tmp/pwiz.version https://teamcity.labkey.org/repository/download/bt36/`cat /tmp/pwiz.build`:id/VERSION?guest=1' returned a non-zero code: 8
```
Please let me know if there is a fix for this or if you need anymore information.

Thanks in advance,

Sat

**Comment by sneumann** [comment]
Should be fixed in 9989b5dc15be0b99f08d6f58ef27576311d2c98e. Yours, Steffen

**Comment by dmarcbriers** [comment]
I am receiving this same error when building this image. I am using the latest master branch of this repo and I get the same error as above:
```
Step 27/38 : RUN wget -O /tmp/pwiz.version https://teamcity.labkey.org/repository/download/bt36/`cat /tmp/pwiz.build`:id/VERSION?guest=1
 ---> Running in 484adfc3c5e3
--2018-11-27 23:44:58--  https://teamcity.labkey.org/repository/download/bt36/606438:id/VERSION?guest=1
Resolving teamcity.labkey.org (teamcity.labkey.org)... 54.245.112.80
Connecting to teamcity.labkey.org (teamcity.labkey.org)|54.245.112.80|:443... connected.
HTTP request sent, awaiting response... 404
2018-11-27 23:44:58 ERROR 404: (no description).
```

The build id in the Dockerfile is "606438"

Any help would be greatly appreciated.
